src/FillArrays.jl

```diff
@@ -33,21 +33,21 @@ abstract type AbstractFill{T, N, Axes} <: AbstractArray{T, N} end
 
 ==(a::AbstractFill, b::AbstractFill) = axes(a) == axes(b) && getindex_value(a) == getindex_value(b)
 
-Base.@propagate_inbounds @inline function _fill_getindex(F::AbstractFill, kj::Integer...)
+Base.@propagate_inbounds function _fill_getindex(F::AbstractFill, kj::Integer...)
     @boundscheck checkbounds(F, kj...)
     getindex_value(F)
 end
```

Comment:
Should this just be `@inbounds`? I thought that a function which deals with `checkbounds` need not propagate anything further, and only functions like `getindex(F::AbstractFill, k::Integer) = _fill_getindex(F, k)` need to do so.

https://docs.julialang.org/en/v1/devdocs/boundscheck/#Eliding-bounds-checks

(But 5 mins of trying to `@eval` things to test didn't tell me anything, so I am not 100% sure.)